jpa/boot-jpa-jooq-sample/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM eclipse-temurin:21.0.5_11-jre-alpine as builder
+WORKDIR application
```

Hadolint (2.12.0)

[error] Use absolute WORKDIR

(DL3000)